.github/workflows/production.yml

```diff
@@ -22,7 +22,7 @@ jobs:
     name: Production Deployment
     uses: ./.github/workflows/deployment.yml
     permissions:
-      contents: read
+      contents: write
```

Comment:
```suggestion
      contents: read
      checks: write
```

Reply:
@pauljohanneskraft Please double check that this is the only permissions mismatch.